sky/clouds/gcp.py

```diff
@@ -216,8 +215,12 @@ def make_deploy_resources_variables(
                 # https://cloud.google.com/compute/docs/gpus
                 resources_vars['gpu'] = 'nvidia-tesla-{}'.format(acc.lower())
                 resources_vars['gpu_count'] = acc_count
-                # CUDA driver version 470.103.01, CUDA Library 11.3
-                image_id = _IMAGE_ID_PREFIX + 'common-cu113'
+                if acc == 'K80':
```

Comment:
acc.upper()? Or would current PR work for `sky gpunode --gpus k80`?

Reply:
Thanks, great catch. in fact this also catches another error when searching acc in region.
```
(sky) weichiang@mbp sky % sky gpunode --gpus k80 --cloud gcp
ValueError: Accelerator "k80" is not available in "us-central1" region/zone.
``` 

But is this still the case after we ship https://github.com/skypilot-org/skypilot/pull/1075? @WoosukKwon

Reply:
Just tested this PR with latest master, it works well with
```
if acc == 'K80':
```

Reply:
Yeah after #1075 the internal name should be always 'K80' regardless of the user input.